docs/Webhooks.md

```diff
@@ -1,179 +1,97 @@
-- `GET` `/v1/sse/{wallet_id}/{topic}/{field}/{field_id}`
-- `GET` `/v1/sse/{wallet_id}/{topic}/{field}/{field_id}/{desired_state}`
+## Implementing Your Event Listener
 
-Valid topics are same as noted above.
+Here's an example of how to implement an event listener using JavaScript:
 
-The `field` and `field_id` in the endpoints above serve as filters, and refers to
-any field in the webhook event payload (excluding `wallet_id` and `topic`). The
-name of the field of interest is passed in `{field}`, and the corresponding ID
-or text that one wants to wait for is passed as `{field_id}`. i.e., you can pass
-`connection_id` as `{field}`, and the ID of the connection you want to see the
-events for as `{field_id}`. This will return only webhook events that match the
-specified filter.
+```javascript
+const EventSource = require('eventsource');
 
-The routes that specify a `{desired_state}` will keep the stream open until a
-single webhook event matches the specified filters, and then return that event.
-The endpoints that don't specify a `{desired_state}`, will keep the stream open
-and return all events matching the topic and optional field/field_id filters.
+const wallet_id = '<your_wallet_id>';
+const url = `http://cloudapi.127.0.0.1.nip.io/tenant-admin/v1/sse/${wallet_id}/proofs/connections/<some_id>/done`;
 
-Here is an example Javascript implementation
+const headers = {
+  'x-api-key': 'tenant.<tenant/wallet_token>',
+};
 
-```js
-const EventSource = require('eventsource');
-url = "http://cloudapi.127.0.0.1.nip.io/tenant-admin/v1/sse/<wallet_id>"
+const eventSource = new EventSource(url, { headers });
 
-const headers = {
-    'x-api-key':"tenant.<tenant/wallet_token>",
-  };
-const eventSource = new EventSource(url,{headers});
 // Event listener for incoming server-sent events
 eventSource.onmessage = (event) => {
   const eventData = JSON.parse(event.data);
-  // log event
-  console.log("EVENT ==> ", eventData)
+  console.log("EVENT ==> ", eventData);
 };
 
 // Event listener for handling errors
 eventSource.onerror = (error) => {
   console.error('EventSource error:', error);
-  // You can add error handling logic here if needed
+  // Add your error handling logic here
 };
 
-console.log("<==============>")
+console.log("Listening for events...");
 ```
+
+This script establishes a connection to the SSE endpoint and listens for incoming events. When an event is received,
+it's logged to the console. Error handling is also implemented to manage any issues that may arise during the connection.
+
+>**_NOTE_**: Ensure that you replace `<your_wallet_id>` with the actual wallet ID and use the appropriate `x-api-key`
+for authentication.
+
+## Authentication
+
+The Waypoint service requires authentication to access the SSE endpoint. This is managed through the `x-api-key` header
+in the request. The key should be in the format `tenant.<tenant/wallet_token>`. Failing to provide valid authentication
+will result in a 403 HTTP Error.
```

Comment:
Let's mention: a tenant can only listen to their own events, and a tenant-admin can listen to all webhooks in their group. I believe it's still set up that way